__tests__/app.test.js

```diff
@@ -212,3 +205,55 @@ describe("GET /api/articles/:articleId", () => {
     });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Return all comments for an article", async () => {
+    const response = await request(app)
+      .get("/api/articles/1/comments")
+      .expect(200);
+    expect(response.body).toHaveLength(11);
```

Comment:
A great test assertion to have, but it could easily be combined with the test below